mls-rs-crypto-webcrypto/src/aead.rs

```diff
@@ -83,9 +83,9 @@ impl Aead {
             .then_some(())
             .ok_or(CryptoError::WrongKeyLength)?;
 
-        let mut params = AesGcmParams::new(key_type.algorithm(), &Uint8Array::from(nonce));
+        let params = AesGcmParams::new(key_type.algorithm(), &Uint8Array::from(nonce));
         let aad = Uint8Array::from(aad.unwrap_or_default());
-        params.additional_data(&aad);
+        params.set_additional_data(&aad);
```

Comment:
bump package version due to this? I'm assuming this is a bug fix?

Reply:
The other method is deprecated, that's all